mobilecoind/src/payments.rs

```diff
@@ -312,6 +312,77 @@ impl<T: UserTxConnection + 'static> TransactionsManager<T> {
         Ok(tx_proposal)
     }
 
+    pub fn generate_tx_from_tx_list(
+        &self,
+        account_key: &AccountKey,
+        input_list: &[UnspentTxOut],
+        receiver: &PublicAddress,
+    ) -> Result<TxProposal, Error> {
+        let fee = MINIMUM_FEE;
```

Comment:
I think it would be better to anticipate setting fees in the future and pass this in with the Request